src/settings.ts

```diff
@@ -554,3 +554,6 @@ export const REACTION_TIMEOUT = 5000; // time in ms
 export const SHOW_EMOJI_IN_REACTION_PAGE = true;
 
 export const ZENDESK_URL_PREFIXES = ["/admin"];
+
+// Max questions number for Poll inside Chat
+export const MAX_QUESTIONS_NUMBER: Number = 8;
```

Comment:
No need to add `_NUMBER` to the end of this. Something like this would probably work better:

```suggestion
export const MAX_POLL_CHOICES: Number = 8;
```